### PR TITLE
Adding quick answers

### PR DESCRIPTION
**Pull request by pscadding**
First pass at adding Quick Answers to the docs (formerly FAQs).

This first pass will not add all of the TK FAQ docs in but does add a few, and puts the hierarchy in place to be built upon.

In time this structure will replace our original FAQ section.

### CODE REVIEW

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://doctest3.s3-website.eu-west-2.amazonaws.com/tk-doc-generator/f19cbc79282ad922a3ef434eff3b3bacb963ea2d/index.html)

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://doctest3.s3-website.eu-west-2.amazonaws.com/tk-doc-generator/b4bd45f06dabb33456f2236c2be29195ec3cb64d/index.html)

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://doctest3.s3-website.eu-west-2.amazonaws.com/tk-doc-generator/ae5f52b5bf51c68fb04fad7da2ccdf5c37107e88/index.html)

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://doctest3.s3-website.eu-west-2.amazonaws.com/tk-doc-generator/c5523d460998a738d32eacfe6acea081d7f2ba96/index.html)

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://doctest3.s3-website.eu-west-2.amazonaws.com/tk-doc-generator/c3d004487911fdb8ced4d49ed4e166e82ae9e2d7/index.html)

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://doctest3.s3-website.eu-west-2.amazonaws.com/tk-doc-generator/127d8b782d7da7e742c46930ea9d19cf30a76066/index.html)

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://doctest3.s3-website.eu-west-2.amazonaws.com/tk-doc-generator/3f307de30c6adf221ef0c853d11146f4a688afbe/index.html)

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://doctest3.s3-website.eu-west-2.amazonaws.com/tk-doc-generator/e87240c475c9968a2e77050b934ec67386e52060/index.html)

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://doctest3.s3-website.eu-west-2.amazonaws.com/tk-doc-generator/110b6d52bffbed820739a3af53aede90ac5e0da8/index.html)

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://doctest3.s3-website.eu-west-2.amazonaws.com/tk-doc-generator/c92fe4d62a987188059812c722570559c4993252/index.html)

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://doctest3.s3-website.eu-west-2.amazonaws.com/tk-doc-generator/07ce5e643ea5fcef59e620b7a8cf16e661a7f83f/index.html)

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://doctest3.s3-website.eu-west-2.amazonaws.com/tk-doc-generator/dbe59a4261670f68bc22b0bfdedba88268b19286/index.html)

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://doctest3.s3-website.eu-west-2.amazonaws.com/tk-doc-generator/5bb0590f6df9afd21a3a5cfa7310544463b4b2f8/index.html)

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://doctest3.s3-website.eu-west-2.amazonaws.com/tk-doc-generator/10def527d257b0489c2c7f12c0dc0c8a35514f64/index.html)

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://doctest3.s3-website.eu-west-2.amazonaws.com/tk-doc-generator/3081e6e6292ef07b17d60714e7621cca70501b38/index.html)

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://doctest3.s3-website.eu-west-2.amazonaws.com/tk-doc-generator/8eaab05c7014612d81abf64b2e27b37486a41956/index.html)

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://doctest3.s3-website.eu-west-2.amazonaws.com/tk-doc-generator/760a1a1d1a6c866e7f4603477adfb323885e4f2b/index.html)

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://doctest3.s3-website.eu-west-2.amazonaws.com/tk-doc-generator/49c4b29c23af33a7458a5f6243113ef78bc07db5/index.html)

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://doctest3.s3-website.eu-west-2.amazonaws.com/tk-doc-generator/6cf53b93943a1a968c9f3f0363523050fc0fbd7b/index.html)

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://doctest3.s3-website.eu-west-2.amazonaws.com/tk-doc-generator/01fb4260f79627e721563efcbbf8fc3bf7d48035/index.html)

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://doctest3.s3-website.eu-west-2.amazonaws.com/tk-doc-generator/e273068fcb113f585030fc49d68fb24f280c315c/index.html)

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://doctest3.s3-website.eu-west-2.amazonaws.com/tk-doc-generator/bbcc1ff8ef7de48d70267795ccfd89b584a7947a/index.html)

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://doctest3.s3-website.eu-west-2.amazonaws.com/tk-doc-generator/11a8a694d54234c57ade5786a0fe48f7f3ab9a8b/index.html)

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://doctest3.s3-website.eu-west-2.amazonaws.com/tk-doc-generator/264d7026e5bb574c26d68b62dc161ceb64714227/index.html)

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://doctest3.s3-website.eu-west-2.amazonaws.com/tk-doc-generator/fed83b6b4e541c9c23c0011f5eea2b636dd4107b/index.html)

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://doctest3.s3-website.eu-west-2.amazonaws.com/tk-doc-generator/55c6fcb0f1022115faec0def6196d10aa0cd5554/index.html)

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://doctest3.s3-website.eu-west-2.amazonaws.com/tk-doc-generator/eb186666a449f2db4c6ab31c097afafced45c8e5/index.html)

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://doctest3.s3-website.eu-west-2.amazonaws.com/tk-doc-generator/3e199ce66a7e49cea5509fedfaa5a5dfcef9f23c/index.html)

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://doctest3.s3-website.eu-west-2.amazonaws.com/tk-doc-generator/12989ce2bb31a14c44a03bc5dd7962693caef114/index.html)

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://doctest3.s3-website.eu-west-2.amazonaws.com/tk-doc-generator/4cf4338325e1204c14357c24aecad60afa93c0b9/index.html)

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://doctest3.s3-website.eu-west-2.amazonaws.com/tk-doc-generator/dc61a3e79bb82be3b374a1e78a0012cb253d32f8/index.html)

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://doctest3.s3-website.eu-west-2.amazonaws.com/tk-doc-generator/9a3246f18a2302875ac62ec9cb65b3c49ba85457/index.html)

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://doctest3.s3-website.eu-west-2.amazonaws.com/tk-doc-generator/368d0add8382b80f81385fcdadf076f6e0aceac5/index.html)

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://doctest3.s3-website.eu-west-2.amazonaws.com/tk-doc-generator/55e0447188cdb02135aa1e74d47825d0cc4e62f0/index.html)

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://doctest3.s3-website.eu-west-2.amazonaws.com/tk-doc-generator/77ee286eb98354bf21d4a36fd8c8161787667c9c/index.html)

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://doctest3.s3-website.eu-west-2.amazonaws.com/tk-doc-generator/a1cf1c5522b62906ee5aae89b9a5eae659415d82/index.html)

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://doctest3.s3-website.eu-west-2.amazonaws.com/tk-doc-generator/b6ff75e7102b9f396d31d97112ee8ba9190b981e/index.html)

File: docs/en/quick-answers/admin/sg_desktop_run_on_ubuntu.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+had relatively poor results. For reference, 
+[you can check out this thread](https://groups.google.com/a/shotgunsoftware.com/d/msg/shotgun-dev/nNBg4CKNBLc/naiGlJowBAAJ).
+
+Please note that we don't have an explicit list of lib dependencies, since Python itself sits on top of a lot of 
+system level libraries.
+
+We don't have official plans for Debian support at the moment. There is the problem of building for Ubuntu, 
+but then there is the need to QA and support the extra operating system as we make changes, which isn't at all trivial.
+
+If you want to run and activate Toolkit manually without Shotgun Desktop (
```
Comment (tannazie) [inline docs/en/quick-answers/admin/sg_desktop_run_on_ubuntu.md]: this open parenthesis is never closed!

File: docs/en/quick-answers/admin/sg_desktop_run_on_ubuntu.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+[as explained in the doc here](https://support.shotgunsoftware.com/entries/95444187) - please 
+download the `activate_shotgun_pipeline_toolkit.py` script from that documentation page - it's in step 8 of the guide,
+ click the "click to download..." header.
```
Comment (tannazie) [inline docs/en/quick-answers/admin/sg_desktop_run_on_ubuntu.md]: Any reason we're not just linking directly to step 8? https://support.shotgunsoftware.com/hc/en-us/articles/219033208#Step%208.%20Run%20the%20activation%20script

File: docs/en/quick-answers/admin/what_is_path_cache.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+permalink: /quick-answers/admin/what_is_path_cache/
+lang: en
+---
+
+What is the Path Cache? What are Filesystem Locations?
+==
+
+The path cache is used by Toolkit to track the associations between folders on disk and entities in Shotgun. 
+The master cache is stored in Shotgun using the `FilesystemLocation` entity type. Each user then has their own version 
+of the path cache [stored locally in the Toolkit cache directory on disk](./where_is_my_cache), which is synchronized in the background
```
Comment (tannazie) [inline docs/en/quick-answers/admin/what_is_path_cache.md]: this link is giving a 404. does it need to be `./where_is_my_cache.md`?

File: docs/en/quick-answers/admin/where_is_my_cache.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
+---
+layout: default
+title: Where is my cache?
+permalink: /quick-answers/admin/where_is_my_cache/
+lang: en
+---
+
```
Comment (tannazie) [inline docs/en/quick-answers/admin/where_is_my_cache.md]: this page is missing a title!

File: docs/en/quick-answers/troubleshooting/houdini_integrations_not_starting.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+for that variable.
+
+For example, if you have something like this already in your houdini.env file:
+
+    HOUDINI_PATH = /example/of/an/existing/path;&
+Then you should add $HOUDINI_PATH; to the end of the path defined in the file and save it:
+
+    HOUDINI_PATH = /example/of/an/existing/path;$HOUDINI_PATH;&
+This will allow the Shotgun set value to persist when Houdini launches.
+
```
Comment (tannazie) [inline docs/en/quick-answers/troubleshooting/houdini_integrations_not_starting.md]: The formatting on these lines of code got messed up in the preview (http://doctest3.s3-website.eu-west-2.amazonaws.com/tk-doc-generator/a1cf1c5522b62906ee5aae89b9a5eae659415d82/quick-answers/troubleshooting/houdini_integrations_not_starting/) – looks like you need a blank line between the line of code and the line of text. (Or just use backticks?)

File: docs/en/quick-answers/admin/sg_desktop_run_on_ubuntu.md (hunk 2 above)
Comment (pscadding) [reply docs/en/quick-answers/admin/sg_desktop_run_on_ubuntu.md]: good point, I just copied from the existing doc, but makes sense that we should

Comment (pscadding) [reply docs/en/quick-answers/admin/sg_desktop_run_on_ubuntu.md]: actually is that doc still going to exist in the future?

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://doctest3.s3-website.eu-west-2.amazonaws.com/tk-doc-generator/9bec2cd7fd56d06c06fe5e50f68fe4ed60d680a5/index.html)

File: docs/en/quick-answers/admin/sg_desktop_run_on_ubuntu.md (hunk 1 above)
Comment (pscadding) [reply docs/en/quick-answers/admin/sg_desktop_run_on_ubuntu.md]: I think I've updated this correctly now

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://doctest3.s3-website.eu-west-2.amazonaws.com/tk-doc-generator/f40b6ace99317b24bea7592cc34476c16a37868f/index.html)

File: docs/en/quick-answers/admin/sg_desktop_run_on_ubuntu.md (hunk 2 above)
Comment (tannazie) [reply docs/en/quick-answers/admin/sg_desktop_run_on_ubuntu.md]: As soon as we merge this, it's live, so ... we should have links that work now.

**Comment by pscadding** [comment]
I'm going to create new pull requests for this to keep the initial commit clean.

**Pull request closed without merging**